packages/jaeger-ui/src/components/Monitor/ServicesView/operationDetailsTable/index.tsx

```diff
@@ -112,9 +112,7 @@ export class OperationTableDetails extends React.PureComponent<TProps, TState> {
               error={error.opsLatencies}
             />
             <div className="table-graph-avg">
-              {typeof value === 'number' && row.dataPoints.service_operation_latencies.length > 0
-                ? formatTimeValue(value * 1000)
-                : ''}
+              {row.dataPoints.service_operation_latencies.length > 0 ? formatTimeValue(value * 1000) : ''}
```

Comment:
This replacement is not completely equivalent. What would be a situation when `value` would not be a number?

Reply:
Hello,

In this case, the value of 'value' is 'ServiceOpsMetrics['latency']', and 'latency' is defined as a number type.
 
 export type ServiceOpsMetrics = {
  dataPoints: OpsDataPoints;
  errRates: number;
  impact: number;
  latency: number;
  name: string;
  requests: number;
  key: number;
};

Therefore, I thought that there is no need for additional type checking because 'value' is always of type number. Am I mistaken?